lib/Cake/Console/Shell.php

```diff
@@ -183,6 +184,7 @@ class Shell extends Object {
  * @link http://book.cakephp.org/2.0/en/console-and-shells.html#Shell
  */
 	public function __construct($stdout = null, $stderr = null, $stdin = null) {
+		Debugger::getInstance()->output('txt');
```

Comment:
Instead of in each shell, could this be done in the ShellDispatcher? That class is responsible for bootstrapping the shell environment/context.